application/node.go

```diff
@@ -273,7 +300,15 @@ func (n *Node) emitNodeEvent(event cc.Command) {
 
 func (n *Node) receiveControllerUpdate(update serialapi.ControllerUpdate) {
 	n.setFromApplicationControllerUpdate(update)
+
+	select {
+	case n.queryStageNIFComplete <- true:
+	default:
+	}
+
+	n.QueryStageNIF = true
 	n.saveToDb()
+	n.nextQueryStage()
```

Comment:
nextQueryStage is the method that runs each step of the interview process. 

Right now the interview process is 4 steps
* get the NIF
* get security cababilities
* get manufacturer info 
* get command classes versions